src/components/common/Select/Select.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import * as Styled from './Select.styled';
+import ChevronDown from '@/assets/svg/chevron-down-16.svg';
+import { useToggleState } from '@/hooks';
+import { ValueOf } from '@/types';
+
+export interface SelectOption {
+  value: string;
+  label: string;
+}
+
+export const SelectSize = {
+  xs: 'xs',
+  sm: 'sm',
+  md: 'md',
+} as const;
+
+export interface SelectProps {
+  size: ValueOf<typeof SelectSize>;
+  value: string;
+  placeholder?: string;
+  options: SelectOption[];
+  setValue: (value: string) => void;
+}
+
+const Select = ({ size, value, placeholder = '전체', options, setValue }: SelectProps) => {
+  const [isOpened, toggleOpened] = useToggleState(false);
+
+  const handleClickOption = (optionValue: string) => {
+    setValue(optionValue);
+    toggleOpened();
+  };
+
+  return (
+    <Styled.SelectContainer>
+      <Styled.Select size={size} onClick={toggleOpened} isOpened={isOpened}>
+        {!value ? (
+          <Styled.SelectPlaceholder>{placeholder}</Styled.SelectPlaceholder>
+        ) : (
+          <Styled.SelectValue>{value}</Styled.SelectValue>
+        )}
```

Comment:
[제안]
```suggestion
        {value ? (
          <Styled.SelectValue>{value}</Styled.SelectValue>
        ) : (
          <Styled.SelectPlaceholder>{placeholder}</Styled.SelectPlaceholder>
        )}
```

!value를 기준으로 삼항연산자를 쓰신 이유가 있을까요?

Reply:
88288b5f6da3c210b56802b50eaecdb29f0cf7c1 수정했습니다.